src/compiler/crystal/codegen/codegen.cr

```diff
@@ -5,7 +5,7 @@ require "../semantic/type_inference"
 require "../program"
 require "./llvm_builder_helper"
 
-LLVM.init_x86
+LLVM.init_all_targets
```

Comment:
Hm... I suppose this part isn't essential for running on FreeBSD...

Will experiment when I work on this next.